xlators/debug/io-stats/src/io-stats.c

```diff
@@ -292,8 +292,10 @@ is_fop_latency_started(call_frame_t *frame)
         begin = &frame->begin;                                                 \
         end = &frame->end;                                                     \
                                                                                \
-        elapsed = gf_tsdiff(begin, end) / 1000.0;                              \
-        throughput = op_ret / elapsed;                                         \
+        elapsed = gf_tsdiff(begin, end);                                       \
+        if (elapsed) {                                                         \
```

Comment:
If `elapsed` is 0, `throughput` won't be initialized.

Reply:
I initialized 'throughput' to 0